### PR TITLE
Update smart contract specs - channel identifier, batch registration

### PR DESCRIPTION
**Pull request by loredanacirstea**
Fixes https://github.com/raiden-network/spec/issues/97

### CODE REVIEW

File: smart_contracts.rst [hunk 1]
```diff
@@ -84,43 +88,108 @@ Provides the interface to interact with payment channels. The channels can only
 
 .. _channel-identifier:
 
-:term:`Channel Identifier` is currently defined as ``keccak256(address participant1, address participant2)``, where the two participant addresses are in lexicographic order.
+:term:`Channel Identifier` is currently defined as ``uint256``, a global monotonically increasing counter of all the channels inside a ``TokenNetwork``.
```
Comment (LefterisJP) [inline smart_contracts.rst]: Please mention here that 0 is not a valid value for the identifier of an active channel.

File: smart_contracts.rst [hunk 2]
```diff
@@ -84,43 +88,108 @@ Provides the interface to interact with payment channels. The channels can only
...
+            bytes32 locksroot,
+            uint256 locked_amount
+        )
+
+- ``channel_identifier``: :term:`Channel identifier` assigned by the current contract.
+- ``participant``: Ethereum address of a channel participant.
+- ``partner``: Ethereum address of the other channel participant.
+- ``deposit``: Can be ``>=0`` after the channel has been opened. Must be ``0`` when the channel is in ``Settled`` or ``Removed`` state.
+- ``withdrawn_amount``: Can be ``>=0`` after the channel has been opened. Must be ``0`` when the channel is in ``Settled`` or ``Removed`` state.
+- ``is_the_closer``: Can be ``true`` if the channel is in ``Closed`` state, if ``participant`` closed the channel. Must be ``false`` otherwise.
```
Comment (LefterisJP) [inline smart_contracts.rst]:
```diff
-state, if ``participant``
+ state and if ``participant``
```

File: smart_contracts.rst (hunk 1 above)
Comment (loredanacirstea) [reply smart_contracts.rst]: I added a note below this definition.

**Comment by LefterisJP** [comment]
Looks good to me then. Thank you Loredana.

**Pull request merged**